<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>duplicate_selector</class>
 <widget class="QWidget" name="duplicate_selector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="MediaViewer" name="mediaviewer" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="radio_buttons">
         <item>
          <widget class="QRadioButton" name="superior_file_radio">
           <property name="text">
            <string>Superior File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="duplicate_radio">
           <property name="text">
            <string>Duplicate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="not_duplicate_radio">
           <property name="text">
            <string>Not Duplicate</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="apply_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Apply</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MediaViewer</class>
   <extends>QWidget</extends>
   <header>src/MediaViewer/mediaviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
